jmx-metrics/README.md

```diff
@@ -68,6 +68,7 @@ mutually exclusive with `otel.jmx.groovy.script`. The currently supported target
 
 | `otel.jmx.target.system` |
 |--------------------------|
+| [`jetty`](./docs/target-systems/jetty.md) |
```

Comment:
It's not a great system, but we have jvm on top and external libs sorted below it.